demo/src/index.css

```diff
@@ -1,3 +1,42 @@
 body {
   font-family: Arial, Helvetica, sans-serif;
+  --foreground-colour: rgb(14, 11, 92);
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+details > summary {
+  cursor: pointer;
```

Comment:
Seems to keep a pointer in the details too 😕

Reply:
Fixed in 81fd52b2816f32d88159044f88314bcaaf74ef85.